107/src/main/java/org/ehcache/jsr107/Unwrap.java

```diff
@@ -20,13 +20,15 @@
  */
 final class Unwrap {
 
-  static <T> T unwrap(Class<T> clazz, Object obj) {
+  static <T> T unwrap(Class<T> clazz, Object... obj) {
     if (clazz == null || obj == null) {
```

Comment:
It was possible in some cases that one of the objects is null, causing NPE. Thus added a check here.